tests/nanoemoji_test.py

```diff
@@ -38,14 +40,26 @@
 )
 
 
-@pytest.fixture(scope="module", autouse=True)
+# https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
+    # execute all other hooks to obtain the report object
+    outcome = yield
+    result = outcome.get_result()
+    setattr(item, result.when + "_result", result)
+
+
+@pytest.fixture()
```

Comment:
If you want to keep this, I believe you need to continue using `autouse=True` like before. We want the fixture to be automatically used (not only when a test function references it explicitly as a parameter). I think right now it is not being triggered.

Reply:
Erk good catch

Reply:
Ha, you are right. And that revealed my hook wasn't quite working as intended. I think it's now happy.